README.md

```diff
@@ -27,165 +27,135 @@ and signs AMP documents as requested by the AMP Cache.
-    you want to sign for multiple domains with different certificates, then run
-    different instances of the packager. We recommend using a different
-    certificate/key pair from your normal web-serving traffic. See the example
-    config file for details.
-  * Must be able to make outgoing connections to `cdn.ampproject.org` on port
-    443.
-  * Should have its stdout redirected to a log file somewhere, probably rotated.
-  * Should not run as superuser.
-
-<!-- TODO(twifkak): Add instructions for getting an API key or service account,
-     after the Transformer API is in place. Maybe make a script that automates
-     it using gcloud. -->
-
-Once you've chosen a setup that meets the above constraints, actual
-configuration is fairly straightforward:
-
-  1. `go get github.com/ampproject/amppackager/cmd/amppkg`
-  2. Move the built `~/go/bin/amppkg` wherever you like.
-  3. Create a packager config file; use `amppkg.example.toml` in this repo as a template.
-  4. Launch with `/path/to/amppkg -config=/path/to/amppkg.toml >>/path/to/amppkg.log`.
-  5. Set up log rotation for `amppkg.log`.
+     If `amppkg.toml` is not in the current working directory, pass
+     `-config=/path/to/amppkg.toml`.
 
 #### Test your config
 
-  1. Run a [Chrome Canary](https://www.google.com/chrome/browser/canary.html),
-     or if on Linux, a [nightly build of
-     Chromium](https://www.chromium.org/getting-involved/download-chromium).
-  2. Navigate to chrome://flags#enable-signed-http-exchange and enable that
-     feature.
-  3. Navigate to your AMP package URL (i.e. the `href` of your
-     `<link rel="amppackage">` for a given page).
-  4. Watch the URL transmogrify!
-
-Optionally, you may pretend to be an AMP Cache:
-
-  1. Use `wget` to download the package and save it as a `foo.htxg` file.
-  2. `go run cmd/amppkg_test_cache/main.go -package=path/to/foo.htxg`
-  3. Ensure the packager is still running; it's needed to serve the certificate.
-  4. Visit `http://localhost:8000/` in the experimental Chromium.
-
-#### How will these web packages be discovered?
-
-The details of that are still being worked out. We have several alternatives in
-mind and want to come up with an answer that's best for the web, including
-crawlers, sites serving packages, sites not serving packages, and package
-caches. If you have any constraints or suggestions, please comment on issue #5,
-or feel free to reach out in private if needed.
+  1. Run Chrome M70 or later (as of 2018-09-18, this is
+     [Beta](https://www.google.com/chrome/beta/) or
+     [Dev](https://www.google.com/chrome/dev/)). On the
+     command-line, pass the following flags:
+     ```
+     --user-data-dir=/tmp/udd
+     --ignore-certificate-errors-spki-list=$(openssl x509 -pubkey -noout -in path/to/fullchain.pem | openssl pkey -pubin -outform der | openssl dgst -sha256 -binary | base64)
+     --enable-features=SignedHTTPExchange
```

Comment:
Maybe enable `AllowSignedHTTPExchangeCertsWithoutExtension` too (`--enable-features=SignedHTTPExchange,AllowSignedHTTPExchangeCertsWithoutExtension`)? That lets Chrome not to reject SXG when cert doen't have CanSignHttpExchanges extension.